src/components/button/button.js

```diff
@@ -33,12 +35,47 @@ Button.prototype.handleKeyDown = function (event) {
   }
 }
 
+/**
+* Add event handler for click
```

Comment:
Slightly pedantic, but this doesn't _add_ the event handler – it is the event handler.